.github/workflows/CI.yml

```
@@ -20,23 +20,26 @@ jobs:
       - run: yarn build
   deploy:
     name: Deploy
-    needs: [build]
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master'
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
       - name: Setup node
         uses: actions/setup-node@v1
         with:
           node-version: '12'
       - run: yarn install
       - run: yarn build
-      - name: Configure GH credentials
-        uses: webfactory/ssh-agent@v0.5.0
+      - name: Set up Cloud DOC
+        uses: google-github-actions/setup-gcloud@master
         with:
-          ssh-private-key: ${{ secrets.R2U_BOT_SSH_KEY }}
-      - run: |
-          git config user.email 'bot@r2u.io'
-          git config user.name 'R2U Bot'
-      - run: yarn deploy
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: config project
+        run: gcloud config set project docs-r2u
```

Projeto ficou `sdk-r2u` né?
```suggestion
        run: gcloud config set project sdk-r2u
```

---

sim

---

q massa essa feature do github

---

Aprendi com o Gabão hoje tbm hahahah